swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
@jimmidyson there could be XSS issues if someone can control the content of a response (i.e. by a malicious API object's content), which then is displayed unescaped in Swagger UI (something  like in https://community.rapid7.com/community/infosec/blog/2016/09/02/r7-2016-19-persistent-xss-via-unescaped-parameters-in-swagger-ui). 

Not that I say that someone can easily control the content we send over our API, but Swagger UI could potentially used for XSS as long as it can show user generated content, which can lead to actions executed in the context of a connected user.

All I wanted to say, that it increases the attack surface, which could be easily avoided by dropping it when not really needed.